src/status_im/contexts/chat/group_details/view.cljs

```diff
@@ -167,4 +167,10 @@
        :render-section-footer-fn       contacts-section-footer
        :render-data                    {:chat-id chat-id
                                         :admin?  admin?}
-       :render-fn                      contact-item-render}]]))
+       :render-fn                      contact-item-render}]
+     [quo/floating-shell-button
+      {:key     :shell
```

Comment:
why key?

(we also have similar :key and :on-press in `status-im.contexts.profile.settings.view`, needs to be refactored)
(will be addressed in https://github.com/status-im/status-mobile/issues/19192)

Reply:
please remove this key, I am not sure why we added in settings page. (And might also show error (component with duplicate key etc...)